lib/serviceClient.ts

```diff
@@ -270,6 +273,18 @@ export class ServiceClient {
         }
       }
 
+      if (operationSpec.responses) {
+        let rawResponse = false;
```

Comment:
this can maybe be more compact with `httpResponse.rawResponse = Object.values(operationSpec.responses).some(spec => ...)` or similar, but it's not super important

Reply:
I'll leave it this way for now, but sure, we can change it later.